content/articles/domains-cm.md

```diff
@@ -0,0 +1,24 @@
+---
+title: .CM Domains
+excerpt: This article explains the requirements and special procedures for .CM domain names.
+categories:
+- Domains
+---
+
+# .CM Domain Names
+
+* TOC
+{:toc}
+
+---
+
+This article explains the requirements and special procedures for .CM domain names.
+
+
+## Domain Restrictions on a .CM domain {#domain-restrictions}
+
+If the name servers of a .CM domain are modified more than three to four times in a month, the domain is set to serverUpdateProhibited for the next four weeks.
+
```

Comment:
Should we also add that .CM domains can't be restored?

https://kb.centralnicreseller.com/domains/tlds/cm

Reply:
that seems... extremely important